src/main/scala/ch/epfl/pop/json/json.scala

```diff
@@ -50,7 +50,12 @@ package object json {
                                    signature: Signature,
                                    message_id: Hash,
                                    witness_signatures: List[Signature]
-                                 )
+                                 ) {
+
+    def updateWitnesses(s: Signature): MessageContent =
+      MessageContent(encodedData, data, sender, signature, message_id, s :: witness_signatures)
```

Comment:
Once this PR will be accepted, I'll change this function and update the code to reflect changes done in commit 5a14e14 (witness_signatures is now a `List[KeySignPair]` which is a pair of public key and corresponding signature)